### DISCUSSION TITLE
Welcome to blazorbootstrap Discussions!

### DISCUSSION THREAD

**Discussion by gvreddy04** [vikramlearning/blazorbootstrap#111] [Q&A]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mvit777** [comment]
Hi,
my name is Marcello. Been playing with the lib for a while and also developed a similar concept some time ago (https://github.com/mvit777/BikeShop/tree/master/BikeShop.BlazorComponents).
I was wondering when datagrid binding occurs during page lifecycle.
thnx

**Reply by gvreddy04** [reply]
@mvit777 DataGrid binding called inside the `OnAfterRenderAsync(bool firstRender)` method.

**Suggestion:** Always create a new discussion. So that we can answer ASAP.

**Reply by mvit777** [reply]
got it
